plugins/uefi-capsule/fu-uefi-common.c

```diff
@@ -244,12 +244,15 @@ fu_uefi_get_esp_path_for_os (FuDevice *device, const gchar *base)
 	if (g_file_test (esp_path, G_FILE_TEST_IS_DIR) || os_release == NULL)
 		return g_steal_pointer (&esp_path);
 	/* if ID key doesn't exist, try ID_LIKE */
-	id_like_id = g_hash_table_lookup (os_release, "ID_LIKE");
-	if (id_like_id != NULL) {
-		g_autofree gchar* id_like_path = g_build_filename (base, "EFI", id_like_id, NULL);
-		if (g_file_test (id_like_path, G_FILE_TEST_IS_DIR)) {
-			g_debug ("Using ID_LIKE key from os-release");
-			return g_steal_pointer (&id_like_path);
+	id_like = g_hash_table_lookup (os_release, "ID_LIKE");
+	if (id_like != NULL) {
+		g_auto(GStrv) split = g_strsplit (id_like, " ", -1);
+		for (guint i = 0; split[i] != NULL; i++) {
+			g_autofree gchar* id_like_path = g_build_filename (base, "EFI", split[i], NULL);
```

Comment:
Can you move the `gchar* id_like_path` to `gchar *id_like_path` -- thanks. I'm aware the original is wrong too :) Ta.

Reply:
Done. Hope that's ok because I had to relearn squashing commits...